### ISSUE TITLE
Setting context properties

### ISSUE THREAD

**Issue by Farre82** [MarkPieszak/angular-application-insights#37]
Setting context properties like version, UserID. Is that possible?

**Comment by MarkPieszak** [comment]
You should be able to, what are you trying to do?
The API is (at least supposed) to be identical, to the normal application-insights.

**Comment by worldspawn** [comment]
userId is set with `setAuthenticatedUserContext`.

To set custom properties i think you need to add a custom telemetry initializer.

https://github.com/Microsoft/ApplicationInsights-JS/blob/master/API-reference.md#addtelemetryinitializer

This doesn't appear to exposed though _this_ library.

**Comment by worldspawn** [comment]
I think the problem is here: 

https://github.com/MarkPieszak/angular-application-insights/blob/b1deeeb27ddc5b1a0eef8208c79e4f2cedb82355/src/app-insight.service.ts#L42-L48

`AppInsightsService` defines a `queue` and `context` field but they are never set to anything.

```
constructor( @Optional() _config: AppInsightsConfig, public router: Router) {
    console.log('service created');
    this.config = _config;
    this.queue = AppInsights.queue;
    this.context = AppInsights.context;
  }
```

FTW?

**Comment by worldspawn** [comment]
Should then be able to do something like this:

```appInsights.queue.push(() => {
      appInsights.context.addTelemetryInitializer((envelope: Microsoft.ApplicationInsights.IEnvelope) => {
        const ti = envelope.data.baseData;

        ti.properties = ti.properties || {};
        ti.properties['foo'] = 'bar';
      });
    });```

**Comment by MarkPieszak** [comment]
Fixed via e5040bea467f2e759583cc9dae8b3f8bc7ec43b5 thanks to @worldspawn 🎖
